modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js

```diff
@@ -32,21 +33,22 @@ const getToolbarSet = (toolbarSet) => {
 const ClassicEditor = ({
 	contents = '',
 	cssClass,
-	editorConfig = {},
+	editorConfig,
 	initialToolbarSet,
 	name,
 	onChangeMethodName,
+	title,
 }) => {
 	const editorRef = useRef();
 
 	const [toolbarSet, setToolbarSet] = useState(initialToolbarSet);
 
-	const config = useMemo(() => {
+	const getConfig = () => {
```

Comment:
> I removed useMemo in config override to simplify, I don't think there is any reason for it.

I can't comment on this, honestly... but I guess we're generating `getConfig` over and over when nothing in it changes... maybe @wincent or @carloslancha have something to say about this.

Reply:
In this case there's unlikely to be any benefit from the `useMemo`.

I would test that assumption by adding a `console.log()` in here just to get a feel for how often it is called.

Regardless of how many times, we still can compare the relative costs:

- Without `useMemo`, we will create a new object every time `getConfig()` is called. If `editorConfig` has a lot of keys, then the cost of spreading them into the new object will grow linearly with the number of keys, but browsers *are* fast. In short, unless this is called very often, we'll end up creating a bit of short-lived(?) garbage at what is probably a low cost.
- With `useMemo`, we avoid the new objects, but pay the cost of a function call, some book-keeping data structure, and a couple of internal calls to `Object.is()`. So, we avoid the memory overhead (copies are made strictly when necessary) but pay a small tax for the memoization overhead.

The reason why I said I'd log to get a sense of how often this is happening. If it is getting called half a dozen times, then the difference between the options is going to be insignificant. And if it is getting called a lot of times, then that might be a separate problem that we want to fix anyway.

(In any case, code call getting rid of the default value for the `editorConfig` param; that could have been a source of pointless memoiziaton invalidation anyway.)

Reply:
On `resize`, we are modifying `toolbarSet` and, consequently, `config`. `getToolbarSet` is called a lot, maybe we should add come debouncing there, but `config` in modified only few times, when phone / tablet size thresholds are passed.

Reply:
So, these are probably roughly the same:

- Copying an object + spreading.
- Calling some functions (`useMemo` plus `Object.is` twice).

But if resizing is involved, debouncing doesn't sound like a bad idea (regardless of whether you're using `useMemo` or not).